config.go

```diff
@@ -141,6 +141,8 @@ type config struct {
 	DropTxIndex        bool          `long:"droptxindex" description:"Deletes the hash-based transaction index from the database on start up and then exits."`
 	AddrIndex          bool          `long:"addrindex" description:"Maintain a full address-based transaction index which makes the searchrawtransactions RPC available"`
 	DropAddrIndex      bool          `long:"dropaddrindex" description:"Deletes the address-based transaction index from the database on start up and then exits."`
+	RelayNonStd        bool          `long:"relaynonstd" description:"Denotes if non-standard transactions should be relayed by btcd."`
```

Comment:
I'm not sure I would figure out the meaning of these from the descriptions.

What do you think of the following?

> Relay non-standard transactions regardless of the default settings for the active network.
> Reject non-standard transactions regardless of the default settings for the active network.